cmake/BuildFlags.cmake

```
@@ -144,6 +144,10 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "MSVC"
   set(SYCL_FLAGS ${SYCL_FLAGS} ${SYCL_KERNEL_OPTIONS})
 
   set(SYCL_OFFLINE_COMPILER_FLAGS "${SYCL_OFFLINE_COMPILER_AOT_OPTIONS}${SYCL_OFFLINE_COMPILER_CG_OPTIONS}")
+
+  message(STATUS "**** SYCL_EXECUTABLE **** ${SYCL_EXECUTABLE}")
+  message(STATUS "**** SYCL_FLAGS **** ${SYCL_FLAGS}")
+  message(STATUS "**** SYCL_OFFLINE_COMPILER_FLAGS **** ${SYCL_OFFLINE_COMPILER_FLAGS}")
```

Pls. follow the style of PyTorch by removing star symbols.

```suggestion
  message(STATUS "SYCL_EXECUTABLE: ${SYCL_EXECUTABLE}")
  message(STATUS "SYCL_FLAGS: ${SYCL_FLAGS}")
  message(STATUS "SYCL_OFFLINE_COMPILER_FLAGS: ${SYCL_OFFLINE_COMPILER_FLAGS}")
```